samples/boot/saml2login/src/main/java/sample/SecurityConfig.java

```diff
@@ -80,17 +80,16 @@ SecurityFilterChain web(HttpSecurity http, LogoutSuccessHandler successHandler,
 					redirect.onLogoutSuccess(request, response, authentication);
 				})
 			)
-			.addFilterAfter(new Saml2RelyingPartyInitiatedLogoutFilter(requestResolver), LogoutFilter.class);
+			.addFilterAfter(new Saml2RelyingPartyInitiatedLogoutFilter(requestResolver()), LogoutFilter.class);
 
 		return http.build();
 	}
 
 	@Bean
-	Saml2AssertingPartyInitiatedLogoutSuccessHandler logoutSuccessHandler(Saml2LogoutResponseResolver requestResolver) {
-		return new Saml2AssertingPartyInitiatedLogoutSuccessHandler(requestResolver);
+	Saml2AssertingPartyInitiatedLogoutSuccessHandler logoutSuccessHandler() {
+		return new Saml2AssertingPartyInitiatedLogoutSuccessHandler(responseResolver());
 	}
 
-	@Bean
```

Comment:
It seems like `requestResolver()` and `requestResolver()` don't need to be beans